src/python/gem5/prebuilt/hifiveunmatched/README.md

```diff
@@ -0,0 +1,52 @@
+# HiFive GEM5 Board Specs and Usage
+
+## Specs for the HiFive Unmatched
+
+### Cache Hierarchy
+- L1D Size : 32kB
+- L1D Associativity : 8
+- L1I Size : 32kB
+- L1I Associativity : 4
+- L2 Size : 2MB
```

Comment:
Note (in theory) this is actually configurable. We may want to make this a parameter of the board.